<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacher</class>
 <widget class="QWidget" name="teacher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>247</width>
     <height>238</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700;&quot;&gt;教师登录窗口&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="id">
        <property name="text">
         <string>编号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="name">
        <property name="text">
         <string>姓名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="sex">
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sexEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="age">
        <property name="text">
         <string>年龄</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ageEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="job">
        <property name="text">
         <string>职称</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="jobEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="getButton">
        <property name="text">
         <string>登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
